# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/mplayer/no-altivec2.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


When altivec is disabled there is no swScale_altivec ...

Additionally cleaned up the else case a bit.

  - Rene Rebe <rene@exactcode.de>

RUN_CPUDETECT='0' failed, cut down to the bare minimum.

  - Dimitar Zhekov <jimmy@is-vn.bg>

--- ./postproc/swscale.c.orig	2004-04-26 22:38:17.000000000 +0300
+++ ./postproc/swscale.c	2004-05-08 12:53:21.000000000 +0300
@@ -1344,10 +1344,11 @@
 
 #else
 #ifdef ARCH_POWERPC
+#ifdef HAVE_ALTIVEC
 	if(flags & SWS_CPU_CAPS_ALTIVEC)
 	  return swScale_altivec;
 	else
-	  return swScale_C;
+#endif
 #endif
 	return swScale_C;
 #endif
